README.md

```diff
@@ -17,40 +18,97 @@ In contrast, our philosophy is rooted in simplicity:
 
 We believe this approach is the best way to build a client that is both fast and resilient. By adhering to these principles, we will be able to iterate fast and explore next-generation features early, either from the Ethereum roadmap or from innovations from the L2s.
 
-## Usage
+## Quick Start (localnet)
+
+![Demo](https://raw.githubusercontent.com/lambdaclass/ethereum_rust/update-readme/Demo.png)
+
+You'll need docker, docker compose, clang and rust 1.80
+
+### Mac
```

Comment:
I would just use the link that you added in the linux part fo everything, since it has instructions for every os.

Reply:
I've unified everything into a single 'Setup' header, wdy?